src/PixiEditor/ViewModels/SubViewModels/Main/MiscViewModel.cs

```diff
@@ -21,6 +22,14 @@ public MiscViewModel(ViewModelMain owner)
     [Command.Basic("PixiEditor.Links.OpenOtherLicenses", "https://pixieditor.net/docs/Third-party-licenses", "Third Party Licenses", "Open Third Party Licenses", IconPath = "Globe.png")]
     public static void OpenHyperlink(string url)
     {
-        ProcessHelpers.ShellExecute(url);
+        try
+        {
+            ProcessHelpers.ShellExecute(url);
+        }
+        catch (Exception e)
+        {
+            NoticeDialog.Show(title: "Error", message: $"Couldn't open the address {url} in your default browser");
+            _ = CrashHelper.SendExceptionInfoToWebhook(e);
```

Comment:
shouldn't it be async? What if someone has poor internet connection? Will it just hang?

Reply:
I'd make async command implementation for that